### PR TITLE
quentin's second haiku

### PR DESCRIPTION
**Pull request by quentingaeschlin**
a thought going through my mind as i look out the window

### CODE REVIEW

File: branch_training2/HaiQu [hunk 1]
```diff
@@ -0,0 +1,3 @@
+the sun is shining
+what am i doing
+in an online meeting?
```
Comment (KristinaPetrle) [inline branch_training2/HaiQu]: i love it, can you please make it longer? add 2 lines please

**Review by KristinaPetrle — CHANGES_REQUESTED** [review]
requested additional lines

File: branch_training2/HaiQu (hunk 1 above)
Comment (quentingaeschlin) [reply branch_training2/HaiQu]: ok understood, will do

Comment (quentingaeschlin) [reply branch_training2/HaiQu]: actually, i ca not do. Andrej should do.

Comment (Pongman007) [reply branch_training2/HaiQu]: done

**Review by KristinaPetrle — APPROVED** [review]

**Pull request merged**